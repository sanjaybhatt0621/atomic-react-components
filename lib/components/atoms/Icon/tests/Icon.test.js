import React from 'react';
import { shallow } from 'enzyme';

import { IconVanilla } from '../index';

describe('<Icon />', () => {
  let IconComponent = '';
  beforeEach(() => {
    IconComponent = shallow(
      <IconVanilla alt="test" path="abcd">
        Test
      </IconVanilla>
    );
  });

  test('should render correctly', () => {
    expect(IconComponent).toMatchSnapshot();
  });
});
