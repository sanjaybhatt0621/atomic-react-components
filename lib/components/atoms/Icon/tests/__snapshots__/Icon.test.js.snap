// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> should render correctly 1`] = `
<svg
  className="icon"
>
  <use />
</svg>
`;
